test/unit/transforms/mapperTransforms.spec.ts

```diff
@@ -132,6 +136,7 @@ describe("transformObjectMapper", () => {
       serializedName: objectName,
       type: {
         name: MapperType.Composite,
+        className: getMapperClassName(schema),
```

Comment:
Maybe we need a string here to express what the `className` should be?  I think this would always pass because it's the same result as what get's put into the mapper, right?

Reply:
Yeah you're right adding the string